src/main.ts

```diff
@@ -1,9 +1,11 @@
+import { ValidationPipe } from "@nestjs/common";
 import { NestFactory } from "@nestjs/core";
 import { AppModule } from "./app.module";
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
   app.enableCors();
+  app.useGlobalPipes(new ValidationPipe({ whitelist: true }));
   await app.listen(process.env.PORT || 8080);
 }
```

Comment:
Do you think we need to add a prefix like "v1" for our API router? If you think it is necessary, we can add "v1" as a prefix.